### PR TITLE
Temp branch

### PR DESCRIPTION
**Pull request by aishbhisikar**

### CODE REVIEW

File: style.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const navToggler = document.querySelector(".myNav-toggler");
+navToggler.addEventListener('click', () => {
+   hideSection();
+   toggleNavbar()
+    document.body.classList.toggle("hide-scrolling")
+});
+
+function hideSection(){
+    document.querySelector("section.active").classList.toggle("fade-out")
```
Comment (luciavelasco) [inline style.js:9]: This is a difficult way to do this. Let me break this down the way I would do it:

When I click on the menu, I want all the sections to be hidden.
When I click on a nav-item or the toggleNavbar button, I want all the sections to be revealed.

Everything else is handled for us in the HTML. Try pseudo-coding that; to do that you will have to change this function to toggle `fade-out` on EVERY section, not just the active one.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+}
+.row{
+    display: flex;
+    flex-wrap: wrap;
+}
+.align-items-center{
+    align-items:center;
+}
+/*custom scrollbar*/
+::-webkit-scrollbar{
```
Comment (luciavelasco) [inline style.css:150]: Overriding the scrollbar properties reduces accessibility, I would recommend removing the scrollbar styles, but if you keep them, I recommend making the width bigger, as 5px is too narrow

Comment (aishbhisikar) [reply style.css:150]: Okay, I'll make it bigger by increasing its width. Thank You.

File: style.js (hunk 1 above)
Comment (aishbhisikar) [reply style.js:9]: I tried but when I did toggleNavbar button stops working. maybe I'm doing something wrong in that.  Can we solve this together on Monday?

**Pull request merged**